docs/release-notes/2.41.1.md

```diff
@@ -26,7 +26,7 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 
 ## Improvements
-
+[PMM-12712](https://jira.percona.com/browse/PMM-12712) -  Added a new metric collection for MongoDB exporters: `mongodb_shards_collection_chunks_count`. Metric is available on sharded Mongo. Use this new metric to assess the distribution of a collection across different shards, and get valuable insights into their load balance.
```

Comment:
@catalinaadam Please take a look on latest changes and feel free to correct it.

Reply:
Looks good! thank you